mk/test.mk

```diff
@@ -15,8 +15,12 @@ export TEST_ASSET_KUBE_APISERVER=$(KUBE_APISERVER_PATH)
 export TEST_ASSET_ETCD=$(ETCD_PATH)
 export TEST_ASSET_KUBECTL=$(KUBECTL_PATH)
 
+TEST_TARGETS ?= test/api test/k8s test/kuma
+
 .PHONY: test
-test: ${COVERAGE_PROFILE} test/api test/k8s test/kuma coverage ## Dev: Run tests for all modules
+test: ${COVERAGE_PROFILE} ## Dev: Run tests for all modules
```

Comment:
I believe you can just do:
```makefile
test: ${COVERAGE_PROFILE} ${TEST_TARGETS} coverage
```
without for-loop

Reply:
Nope, that does not work. Ping me offline to explain, but essentially when there is invocation to the same target within a single `make` execution, it will not execute the target only once.

Reply:
The trick is that you can't. Current implementation did not run `test/k8s` because `test/module` target was already run with `test/api`